features/dd-sdk-android-session-replay/src/main/kotlin/com/datadog/android/sessionreplay/internal/processor/RumContextData.kt

```diff
@@ -10,6 +10,5 @@ import com.datadog.android.sessionreplay.internal.utils.SessionReplayRumContext
 
 internal data class RumContextData(
     internal val timestamp: Long,
-    internal val newRumContext: SessionReplayRumContext,
-    internal val prevRumContext: SessionReplayRumContext
+    internal val newRumContext: SessionReplayRumContext
```

Comment:
I guess after removal of `prevRumContext` having the naming `newRumContext` can be a bit confusing. Maybe this class should be called `TimedSessionReplayRumContext`? it seems it carries only `SessionReplayRumContext` and `timestamp` after all. WDYT? Not pushing for it anyway.

Reply:
👍 I'll rename this to `RecordedQueuedItemContext`